sdk/monitor/monitor-ingestion/review/monitor-ingestion.api.md

```diff
@@ -25,25 +35,9 @@ export interface UploadLogsError {
 }
 
 // @public
-export interface UploadLogsOptions {
+export interface UploadLogsOptions extends OperationOptions {
+    errorCallback?: (uploadLogsError: UploadLogsError) => void;
```

Comment:
I wonder if we should call this `onError` to align with `onResponse` which exists in OperationOptions.

Reply:
or `onErrorResponse` if we wanted to be a little more verbose

Reply:
Renamed to onError